tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
```

Comment:
Do we still do `"extends": "@tsconfig/node18/tsconfig.json"` or is that out of practice?

Reply:
Just an oversight, added in https://github.com/Automattic/vip-design-system/pull/224/commits/5f560f6f3a6114a668b8441383e1424623e52189

Thanks!